Patterns/English/English-PhoneNumbers.yaml

```diff
@@ -0,0 +1,5 @@
+---
+NumberReplaceToken: '@builtin.phonenumber'
+ForbiddenPrefixRegex: !simpleRegex
+  def: (account|card)(\s+(#|number))?:?\s*$
```

Comment:
In the specs I mentioned a couple other cases we want to filter, so the regexes would need to be tweaked.